[2023-12-11 08:09:25,979] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_postgres.crear_tabla_postgres scheduled__2023-12-05T00:00:00+00:00 [queued]>
[2023-12-11 08:09:26,049] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_postgres.crear_tabla_postgres scheduled__2023-12-05T00:00:00+00:00 [queued]>
[2023-12-11 08:09:26,051] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-11 08:09:26,052] {taskinstance.py:1377} INFO - Starting attempt 3 of 6
[2023-12-11 08:09:26,053] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-11 08:09:26,135] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): crear_tabla_postgres> on 2023-12-05 00:00:00+00:00
[2023-12-11 08:09:26,141] {standard_task_runner.py:52} INFO - Started process 1530 to run task
[2023-12-11 08:09:26,153] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_con_conexion_postgres', 'crear_tabla_postgres', 'scheduled__2023-12-05T00:00:00+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/dag_postgres_database.py', '--cfg-path', '/tmp/tmp_dildskn', '--error-file', '/tmp/tmp30hd_w93']
[2023-12-11 08:09:26,156] {standard_task_runner.py:80} INFO - Job 93: Subtask crear_tabla_postgres
[2023-12-11 08:09:26,413] {task_command.py:371} INFO - Running <TaskInstance: dag_con_conexion_postgres.crear_tabla_postgres scheduled__2023-12-05T00:00:00+00:00 [running]> on host 03477fa6e3b6
[2023-12-11 08:09:26,855] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=dag_con_conexion_postgres
AIRFLOW_CTX_TASK_ID=crear_tabla_postgres
AIRFLOW_CTX_EXECUTION_DATE=2023-12-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-05T00:00:00+00:00
[2023-12-11 08:09:26,906] {base.py:68} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-12-11 08:09:27,008] {dbapi.py:231} INFO - Running statement: 
            create table if not exists fin_mundo(
                dt date,
                pais varchar(30)
            )
        , parameters: None
[2023-12-11 08:09:27,023] {postgres.py:94} INFO - NOTICE:  la relación «fin_mundo» ya existe, ignorando

[2023-12-11 08:09:27,074] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_con_conexion_postgres, task_id=crear_tabla_postgres, execution_date=20231205T000000, start_date=20231211T080925, end_date=20231211T080927
[2023-12-11 08:09:27,133] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-11 08:09:27,261] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
